examples/pipeline_tacotron2/inference.py

```diff
@@ -0,0 +1,294 @@
+        default='english_characters',
+        type=str,
+        choices=available_symbol_set,
+        help='select text preprocessor to use.'
+    )
+    preprocessor.add_argument(
+        '--phonemizer', 
+        default="DeepPhonemizer",
+        type=str,
+        choices=available_phonemizers,
+        help='select phonemizer to use, only used when text-preprocessor is "english_phonemes"'
+    )
+    preprocessor.add_argument(
+        '--phonemizer-checkpoint',
+        default="./en_us_cmudict_forward.pt",
+        type=str,
+        help='the path or name of the checkpoint for the phonemizer, '
+             'only used when text-preprocessor is "english_phonemes"'
+    )
+    preprocessor.add_argument(
+        '--cmudict-root',
+        default="./",
+        type=str,
+        help='the root directory for storing CMU dictionary files'
+    )
+
+    audio = parser.add_argument_group('audio parameters')
+    audio.add_argument(
+        '--sample-rate',
+        default=22050,
+        type=int,
+        help='Sampling rate'
+    )
+    audio.add_argument(
+        '--n-fft',
+        default=1024,
+        type=int,
+        help='Filter length for STFT'
+    )
+    audio.add_argument(
+        '--n-mels',
+        default=80,
+        type=int,
+        help=''
+    )
+    audio.add_argument(
+        '--mel-fmin',
+        default=0.0,
+        type=float,
+        help='Minimum mel frequency'
+    )
+    audio.add_argument(
+        '--mel-fmax',
+        default=8000.0,
+        type=float,
+        help='Maximum mel frequency'
+    )
+
+    # parameters for WaveRNN
+    wavernn = parser.add_argument_group('WaveRNN parameters')
+    wavernn.add_argument(
+        '--wavernn-checkpoint-name',
+        default="wavernn_10k_epochs_8bits_ljspeech",
+        choices=list(_MODEL_CONFIG_AND_URLS.keys()),
+        help="Select the WaveRNN checkpoint."
+    )
+    wavernn.add_argument(
+        "--wavernn-loss",
+        default="crossentropy",
+        choices=["crossentropy"],
+        type=str,
+        help="The type of loss the WaveRNN pretrained model is trained on.",
+    )
+    wavernn.add_argument(
+        "--wavernn-no-batch-inference",
+        default=False,
+        action="store_true",
+        help="Don't use batch inference for WaveRNN inference."
+    )
+    wavernn.add_argument(
+        "--wavernn-no-mulaw",
+        default=False,
+        action="store_true",
+        help="Don't use mulaw decoder to decode the signal."
+    )
+    wavernn.add_argument(
+        "--wavernn-batch-timesteps",
+        default=11000,
+        type=int,
+        help="The time steps for each batch. Only used when batch inference is used",
+    )
+    wavernn.add_argument(
+        "--wavernn-batch-overlap",
+        default=550,
+        type=int,
+        help="The overlapping time steps between batches. Only used when batch inference is used",
+    )
+
+    return parser
+
+
+def unwrap_distributed(state_dict):
```

Comment:
I feel we should update the pre-trained model, so that the structure is unwrapped, if that's not BC-breaking. Thoughts?

Reply:
For this part, it is used to load the checkpoints after the user train a model with `train.py`. So I think we can keep it this way (assuming the user trained the model from scratch themselfves.

To load our pertained model build in the library, I'll create a separate argument for it.

Reply:
I've created an issue [here](https://github.com/pytorch/audio/issues/1713) for future improvement.